### DISCUSSION TITLE
Problem with display pixel-perfect

### DISCUSSION THREAD

**Discussion by sonht-0555** [thenick775/gbajs3#140] [General]
@thenick775 
<img width="1867" alt="11111" src="https://github.com/thenick775/gbajs3/assets/26568422/310124f0-86d1-45a3-a41e-4c6a859e153d">

I'm seeing a problem with the emulator display, it doesn't really display each pixel correctly, most likely because the screen ratio is not correct 240x160px. And iOS does not recognize pixels as odd numbers, so pixels are being rounded to integers. 

This [article](https://stackoverflow.com/questions/47696956/display-pixel-perfect-canvas-on-all-devices) is talking about it quite clearly. This will be something that will take a lot of time to fix. As tested with SkyEmu, it displays pixels in odd numbers quite well.

**Comment by thenick775** [comment]
From the article you linked this would most likely need to be done in the mGBA core, but also potentially in the UI. Looks like there is no solution in the current browser landscape.

I don't think I have the bandwidth to attempt to work on this, and probably do not have the browser skills to make it happen (yet).

Feel free to open pr(s) to take a stab at it if your more familiar in this domain!
